docs/flow.md

```diff
@@ -1117,7 +1117,7 @@ as such, there is a family of flattening operators on flows.
 
 #### flatMapConcat
 
-Concatenating mode is implemented by [flatMapConcat] and [flattenConcat] operators. They are the most direct
+Concatenating mode is implemented by [map] and [flattenConcat] operators. They are the most direct
```

Comment:
The original text was correct. Why do you think it needs to be changed?

Reply:
I think it is a description of `flatMapConcat`. According to the description of `flatMapConcat`, it is a shortcut for `map().flattenConcat(). 
So, I suggested the PR.
If not, what is the original context for?

Reply:
It is a description of the "concatenating mode", which is available when you use `flatMapConcat` and `flattenConcat`.

Reply:
Thanks for your explanation. Let me exclude the wrong changes from the pr.